pyannote/audio/labeling/base.py

```diff
@@ -96,7 +96,7 @@ def to_disk(self, architecture=None, weights=None, overwrite=False):
             self.labeling_.save_weights(weights, overwrite=overwrite)
 
     def fit(self, input_shape, design_labeling, generator,
-            samples_per_epoch, nb_epoch, optimizer='rmsprop', log_dir=None):
+            samples_per_epoch, nb_epoch, loss='categorical_crossentropy', optimizer='rmsprop', log_dir=None):
```

Comment:
Can you please try to follow PEP8 as much as possible? 
In particular, I think this line is a bit long: https://www.python.org/dev/peps/pep-0008/#maximum-line-length